ecr/templates/consignment_export_policy.json.tpl

```diff
@@ -0,0 +1,19 @@
+{
+  "Version": "2008-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowPullConsignmentAPI",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::${intg_account}:role/TDRConsignmentExportECSExecutionRoleIntg"
```

Comment:
This is run against the management account. It says that the roles in each of the three accounts are allowed to access this repository. The problem is that the roles need to exist otherwise you get an error. I've deployed all of my changes on staging so the staging role exists and have added it into this policy. We'll need to add the prod role in manually when we deploy the prod environment.

Reply:
Is it possible to add the creation of these roles to the tdr-terraform-backend to create? Otherwise, maybe this should be added to the documentation here, about creating these roles: https://github.com/nationalarchives/tdr-dev-documentation/blob/master/manual/tdr-create-aws-instructure-setup.md

Reply:
I'll add something to the documentation. The roles are tied quite tightly to the ECS tasks so I don't want to create them in the backend.